1e/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d86565ae-ca60-45c2-a0b9-944ec1e05a15",
+  "app_id": "one-e",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your 1E product metrics to Datadog.",
+    "title": "1E Integration",
+    "media": [],
```

Comment:
I believe an email was sent about this, but we require quality screenshots and/or videos for your listing, so if you could please add some media items as well then we can move forward with releasing your integration. 

Please let me know if you have any questions!

Reply:
I have added the images

Reply:
Thanks! Looks good. I left one last comment, and then it should be good on my end :)